hack/jenkins/test-dockerized.sh

```diff
@@ -21,7 +21,12 @@ set -o xtrace
 
 retry() {
   for i in {1..5}; do
-    "$@" && return 0 || sleep "${i}"
+    if "$@"
```

Comment:
typically we put if and then on the same line: `if "$@"; then`

Reply:
YES, has been updated.